docs/general/builtins/scalar.rst

```diff
@@ -1393,6 +1393,37 @@ See below for an example::
     +-------------+
     SELECT 1 row in set (... sec)
 
+.. _scalar-trunc:
+
+``trunc(number[, precision])``
+------------------------------
+
+`number` is the ``numeric`` value to be truncated.
+`precision` is an ``integer`` indicating the number of decimal places to
+preserve (defaults to 0).
+
+Returns the value truncated to the specified precision. When the precision
+is not specified, or when it is 0, it returns an ``integer``, or ``bigint``.
+When it is specified, the return value will be of type ``double precision``.
```

Comment:
the first sentence here is a good candidate for what I have suggested in my previous comment

you could lift the first sentence out and put it as the first para and use the argument literals. then, in this para, just talk about what the function returns

check out the rest of the document for examples where we've done this previously